buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -1239,6 +1243,7 @@ private void createConfiguration() {
         baseConfig.put("path.logs", confPathLogs.toAbsolutePath().toString());
         baseConfig.put("path.shared_data", workingDir.resolve("sharedData").toString());
         baseConfig.put("node.attr.testattr", "test");
+        baseConfig.put("node.attr.availabilityzone", zone);
```

Comment:
we should put config only for multi zone clusters . 
Instead of passing this , can we use `project.findProperty` instead here ?

Reply:
Sure .. Let me try that.

Reply:
Done